lib/web_ui/lib/src/engine/clipboard.dart

```diff
@@ -91,9 +91,49 @@ class ClipboardAPIPasteStrategy implements PasteFromClipboardStrategy {
 class ExecCommandCopyStrategy implements CopyToClipboardStrategy {
   @override
   void setData(String text) {
-    // TODO(nurhan): https://github.com/flutter/flutter/issues/48578
-    print('Clipboard is only implemented for browsers '
-        'supporting Clipboard API. Use context menu for text editing.');
+    final html.TextAreaElement tempTextArea = _appendTemporaryTextArea();
+    tempTextArea.text = text;
```

Comment:
Shouldn't this be `tempTextArea.value = text;` ?

Reply:
this is the one I am trying to use: https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent

I can probably change it to the value.

Do you think there is a difference?

Reply:
done.